src/org.hdfgroup.hdfview/hdf/view/TreeView/DefaultTreeView.java

```diff
@@ -2871,6 +2871,58 @@ public DataView showDataContent(HObject dataObject) throws Exception {
         return theView;
     }
 
+    /**
+     * Displays the meta data of a data object.
+     *
+     * @param dataObject
+     *            the data object
+     *
+     * @return the MetaDataView that displays the MetaData of the data object
+     *
+     * @throws Exception if a failure occurred
+     */
+    @Override
+    public MetaDataView showMetaData(HObject dataObject) throws Exception {
```

Comment:
Where is this used at? It appears to mostly be doing the same thing as the method it's overriding from the main HDFView class.

Reply:
Well it was added as an abstract to TreeView so the implementation is added.
Interestingly, all the calls reference HDFView version.

Reply:
Looks unnecessary.